packages/fxa-payments-server/src/routes/Subscriptions/Cancel/en.ftl

```diff
@@ -11,3 +11,6 @@ sub-item-cancel-msg =
 sub-item-cancel-confirm =
     Cancel my access and my saved information within
     { $name } on { $period }
+
+# $promotion_name (String) - The name of the promotion.
+sub-promo-coupon-applied = { $promotion_name } coupon applied: <priceDetails />
```

Comment:
This might need some review by @LZoog - I've never personally seen an html tag be used as a placeholder for text. From a localizer point of view, I'd expect something like `{ $price_details }` to understand that the text would be replaced with another string. 

This would also need a comment indicating what the placeholder would be replaced with (i.e. an example of the text that comes after).

Reply:
I'll update to add an example of the text that comes after.

@LZoog - for more context, `{ $price_details }` could be any of the following IDs: `price-details-tax-${interval}`, `price-details-no-tax-${interval}`, `price-details-tax` , or `price-details-no-tax`. The id to use is determined by the `<PriceDetails />` component.

Reply:
@bcolsson We do this for a few elements like`connect-another-find-fx-mobile` for example, see [the FTL](https://github.com/mozilla/fxa/blob/75439904d0bca0f9b70d95639332836266573098/packages/fxa-settings/src/components/Settings/ConnectAnotherDevicePromo/en.ftl#L4-L5) and then [the code](https://github.com/mozilla/fxa/blob/75439904d0bca0f9b70d95639332836266573098/packages/fxa-settings/src/components/Settings/ConnectAnotherDevicePromo/index.tsx#L35-L36), and some others if you search the codebase for `elem={{`. I think we usually use this with text inside of it so it looks a little funky (I'm not sure if self-closing is OK or not, does it have to be `<priceDetails></priceDetails>`?)  but if `<priceDetails />` is what determines and displays the right text and is self-contained, should that even be included in this `sub-promo-coupon-applied`? 🤔 I see on the other side of [this ternary](https://github.com/mozilla/fxa/pull/17138/files#diff-8f7f03056f38b3b0a0a670b3cf0760930bfabd31f073fbd08f8f0366db870c28R251), we use `PlanDetails` by itself.

Reply:
The text being inside the tags is the factor here. As a localizer I need to be able to understand how the string will ultimately be displayed so I can translate it grammatically and naturally. I'm primed to understand that `{ $promotion_name }` will be text, but not `<priceDetails />`.  If it's possible to change `<priceDetails />` into a placeholder that a localizer would understand would become text (e.g. `{ $price_details }`), it would allow localizers to translate this without any confusion.

If this sentence and the price details weren't being combined into a single string we could do something simpler like keeping the string as only ` { $promotion_name } coupon applied:` and indicating what follows below.

Reply:
In [my latest commit](https://github.com/mozilla/fxa/pull/17138/commits/60c85399e06cd9a85a389e78471aa9dc961c15e0), I updated the string to `{ $promotion_name } coupon applied: <priceDetails>%S</priceDetails>` and added more details similar to the note in the [best practices that @bcolsson previously shared](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html), as well as examples of the string to help translators. Please let me know if this works or whether additional revisions are needed. Thanks all! :)